packages/react-native/src/theme/tokens.ts

```diff
@@ -0,0 +1,55 @@
+import { ComponentStyles } from './types';
+import { ColorValue } from '@aws-amplify/ui';
+
+interface BaseTokens {
+  colors: Partial<Colors>;
+}
+
+export interface Tokens extends BaseTokens {
+  components: ComponentStyles;
+}
```

Comment:
Why are these separate from the other types? Seems like they could just live in _types.ts_?

Reply:
Moved